src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,57 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static int splitAndSum(String text) {
+
+        if(checkNullOrEmpty(text)){
+            return 0;
+        }
+        int result = calculator(text);
+        return result;
+
+    }
+
+    private static boolean checkNullOrEmpty(String text) {
+        if(text == null) { return true;}
+        if(text.isEmpty()) { return true;}
```

Comment:
```suggestion
        if (text == null) {
            return true;
        }
        
        if (text.isEmpty()) {
            return true;
        }
```

블록의 경우 개행해서 사용해주세요 :)

Reply:
추가로 해당 메서드의 로직은 간단하게 처리할 수 있을 것 같아요!!

```java
return text == null || text.isEmpty();
```